kubernetes/cray-ims/templates/post_upgrade_hook.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: ims-post-upgrade-hook1-container
-        image: "baseos/alpine:3.13"
+        image: "artifactory.algol60.net/docker.io/alpine:latest"
```

Comment:
Does this need to be included in the chart annotations?

Reply:
Good catch. I'll ask Zach. But, seem like it should.